### ISSUE TITLE
 AWS RDS DB option groups failed with error InvalidParameterCombination: OEM_AGENT option requires a version number.

### ISSUE THREAD

**Issue by sriharsha29** [terraform-providers/terraform-provider-aws#3841]
Hi while creating db option group got the following error with version field.

with version field
Error: Error applying plan:
1 error(s) occurred:
* aws_db_option_group.test: 1 error(s) occurred:
* aws_db_option_group.test: Error modifying DB Option Group: InvalidParameterValue: We do not support setting: version for OEM_AGENT

without version field
aws_db_option_group.test: Error modifying DB Option Group: InvalidParameterCombination: OEM_AGENT option requires a version number. Valid versions are 12.1.0.4.v1,12.1.0.5.v1,13.1.0.0.v1,13.2.0.0.v1,13.2.0.0.v2

Snippet
```
{
    "resource": {
        "aws_db_option_group": {
            "test": {
                "engine_name": "oracle-ee",
                "major_engine_version": "12.1",
                "name": "option-group",
                "option": [
                    {
                        "option_name": "OEM_AGENT",
                        "option_settings": [
                         {
                         "name": "OMS_HOST",
                         "value": "xxx"
                        },
                        {
                         "name": "OMS_PORT",
                         "value": "xxx"
                        },
                        {
                         "name": "AGENT_REGISTRATION_PASSWORD",
                         "value": "xxxx"
                        },
                        {
                            "name": "version",
                            "value":"12.1.0.4.v1"
                        }
                       
                    ],
                        "port": "xxxx",
                        "vpc_security_group_memberships": ["sg-xxxx","sg-xxxxx"]
                   
                    }
                
                ],

                "option_group_description": "Option Group"
            }
        }
    }
}
```

Terraform Version : 
Terraform v0.11.4
+ provider.aws v1.11.0

 resource:
- aws_db_option_group

**Comment by bflad** [comment]
Hi @sriharsha29 👋  Sorry you're running into trouble here. It seems both those errors are being returned from the AWS RDS API. I'm not directly familiar with the Oracle Enterprise Manager configuration for RDS, but maybe its looking for the version to be set via `AGENT_VERSION` and not `version`? 

```
                        {
                            "name": "AGENT_VERSION",
                            "value": "12.1.0.4"
                        }
```

Reference: https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Oracle.Options.OEMAgent.html#Oracle.Options.OEMAgent.Options

**Comment by sriharsha29** [comment]
:( not works 
error: Error applying plan:

1 error(s) occurred:

* aws_db_option_group.test: 1 error(s) occurred:

* aws_db_option_group.test: Error modifying DB Option Group: InvalidParameterValue: We do not support setting: AGENT_VERSION for OEM_AGENT
	status code: 400,
and got some other  issues this time with vpc_security_group_memberships : ["sg-xx","sg-xx"]
1 error(s) occurred:

* aws_db_option_group.test: 1 error(s) occurred:

* aws_db_option_group.test: Error modifying DB Option Group: InvalidParameterValue: Invalid security group , groupId= sg-xxxxxx,sg-xxxxxxxx, groupName=.
	status code: 400

**Comment by sriharsha29** [comment]
In RDS option group for oem_agent required an OptionVersion look like the key is missing in terraform as of  I know it is not option settings it should be an option.

**Comment by bflad** [comment]
Support for option `version` configuration has been merged into master (via #2590) and will release with v1.15.0 of the AWS provider, likely mid next week. 🎉
